.editorconfig

```diff
@@ -0,0 +1,18 @@
+# https://EditorConfig.org
+
+# top-most EditorConfig file
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
```

Comment:
```suggestion
```

On my Windows machine, the default seems to be `crlf`, and git translates the line endings automatically when checking files in, so setting this to `lf` locally does nothing but increase the number of errors that are found by `editorconfig-checker` because all line endings differ from the setting then. :)

Therefore I'd suggest keeping this unspecified.

Reply:
👍 

Node-development on Windows 😲 The bad filesystem-perf is why I need the devcontainer (running Docker Desktop on Windows).